### PR TITLE
one user test success

### PR DESCRIPTION
**Pull request by YoustinaZak**
the file contains:
- treeNode class
- function totree to convert a valid corrected xml file to tree structure
- - function to display tree content

please, keep in mind that the test was done on one user only

### CODE REVIEW

File: XMLtoTree.cpp [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+class treeNode{
+    public :
+    string identifier;
+    string content;
+    treeNode * parent; // for navigating back from child to parent
+    vector <treeNode *> children;
+    long long level;
+
+    treeNode()
```
Comment (mark-magdy) [inline XMLtoTree.cpp:18]: do you plan to insert initializations in the constructor?

**Review by mark-magdy — CHANGES_REQUESTED** [review]
hiiii

File: XMLtoTree.cpp (hunk 1 above)
Comment (YoustinaZak) [reply XMLtoTree.cpp:18]: Left for future improvements

**Comment by YoustinaZak** [comment]
the function has been edited

**Review by mark-magdy — CHANGES_REQUESTED** [review]
1) return vector of roots 
2) use this xml for testing code "<users> <a><c>00</c></a> <b></b> </users> <new> <A> </A> </new>"

**Pull request closed without merging**